lib/genome/importers/pharmgkb/new_pharmgkb.rb

```diff
@@ -0,0 +1,94 @@
+require 'genome/online_updater'
+
+module Genome; module Importers; module Pharmgkb;
+  class NewPharmgkb < Genome::OnlineUpdater
+    attr_reader :file_path, :source
+
+    def initialize(file_path)
+      @file_path = file_path
+    end
+
+    def get_version
+      source_db_version = Date.today.strftime("%d-%B-%Y")
+      @new_version = source_db_version
+    end
+
+    def import
+      remove_existing_source
+      create_new_source
+      create_interaction_claims
+    end
+
+    private
+    def remove_existing_source
+      Utils::Database.delete_source('PharmGKB')
+    end
+
+    def create_new_source
+      @source ||= DataModel::Source.create(
+          {
+              base_url:          'http://www.pharmgkb.org',
+              site_url:          'http://www.pharmgkb.org/',
+              citation:          "From pharmacogenomic knowledge acquisition to clinical applications: the PharmGKB as a clinical pharmacogenomic biomarker resource. McDonagh EM, Whirl-Carrillo M, Garten Y, Altman RB, Klein TE. Biomark Med. 2011 Dec;5(6):795-806. PMID: 22103613",
+              source_db_version:  get_version,
+              source_type_id:    DataModel::SourceType.INTERACTION,
+              source_db_name:    'PharmGKB',
+              full_name:         'PharmGKB - The Pharmacogenomics Knowledgebase'
```

Comment:
Since this overlaps with #396, please add a `license_link` and `license_text` field to this hash.

Reply:
Added both of these fields.